index_twitter_stream.py

```diff
@@ -17,13 +17,20 @@
 from esconn import esconn
 es = esconn()
 
-# Update with topics to follow
-if sys.argv[1]:
-    f = open(sys.argv[1])
+if len(sys.argv) == 2:
```

Comment:
I think I would be in favor of doing `if len(sys.argv) > 1`, since we could eventually have more than 2 CLI params being passed  As mentioned I think argparse is the way to go but this solution will work for now. Thoughts?